components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/model/FilterQueryBuilder.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.organization.management.service.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Create filter query builder.
+ */
+public class FilterQueryBuilder {
+
+    private Map<String, String> stringParameters = new HashMap<>();
+    private int count = 1;
+    private String filter;
+
+    public Map<String, String> getFilterAttributeValue() {
+
+        return stringParameters;
+    }
+
+    public void setFilterAttributeValue(String placeholder, String value) {
```

Comment:
add method comments to public methods. Fix other places as well

Reply:
Comments added via 6534bc4a65f7a5c7396abda24169fb0e5690702a